src/Microsoft.Health.Fhir.Tests.Common/TestFiles/R4/Bundle-ChainingAndSortSearchValidation.json

```diff
@@ -0,0 +1,1966 @@
+{
+    "resourceType": "Bundle",
+    "id": "1eb6b09c8316564abe35fad9b948e0d3",
+    "type": "transaction",
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "PractitionerRole",
+                "id": "fake-practitionerRole-01",
+                "meta": {
+                    "versionId": "3",
+                    "lastUpdated": "2022-05-23T19:31:44.596+00:00"
+                },
+                "active": true,
+                "practitioner": {
+                    "reference": "Practitioner/fake-practitioner-01",
+                    "display": "Dr. Yury"
+                },
+                "organization": {
+                    "reference": "Organization/ef58ea08-d883-3957-8300-150554edc8fb",
+                    "display": "ABC health"
+                },
+                "specialty": [
+                    {
+                        "coding": [
+                            {
+                                "system": "http://snomed.info/sct",
+                                "code": "408443003",
+                                "display": "General medical practice"
+                            }
+                        ],
+                        "text": "General medical practice"
+                    }
+                ],
+                "location": [
+                    {
+                        "reference": "Location/1ee7e785-1991-34c3-9d64-8bed2e20233a",
+                        "display": "ABC health"
+                    }
+                ],
+                "healthcareService": [
+                    {
+                        "reference": "HealthcareService/7f6b44f8-97e1-4e8b-af36-c5edae08c7ea",
+                        "display": "Appointment Requests"
+                    }
+                ]
+            },
+            "request": {
+                "method": "POST",
```

Comment:
Should these be PUTs? I'm concerned about the wrong number of resources being present if the tests are run mulitple times.